#
# (c) Copyright 2013-2017 Hewlett Packard Enterprise Development Company LP
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may
# not use this file except in compliance with the License. You may obtain
# a copy of the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations
# under the License.
#
msgid ""
msgstr ""
"Project-Id-Version: eon 8.0.0.20\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-03-25 22:18-0600\n"
"PO-Revision-Date: 2014-03-25 22:18-0600\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: zh <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: eon/api/middleware/context.py:17
msgid "Role used to identify an authenticated user as administrator."
msgstr "用于将经过验证的用户指定为管理员的角色。"

#: eon/api/middleware/context.py:84
msgid "Invalid service catalog json."
msgstr "无效的服务目录 JSON。"

#: eon/api/v1/esx_proxy_ip.py:120
#, fuzzy, python-format
msgid "Unable to display the ip details.got the error : %s"
msgstr "无法显示 vCenter 详细信息。出现错误: %s"

#: eon/api/v1/esx_proxy_ip.py:135
msgid "Failed to delete. IP associated with esx proxy"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:29
#, python-format
msgid "'%s' contains whitespace"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:40
#, python-format
msgid "'%s' is not a valid IP address"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:54
#, python-format
msgid ""
"Invalid input. '%(target_dict)s' must be a dictionary with keys: "
"%(expected_keys)s"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:65
#, python-format
msgid ""
"Validation of dictionary's keys failed.Expected keys: %(expected_keys)s "
"Provided keys: %(provided_keys)s"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:79
#, python-format
msgid "Invalid data format for IP pool: '%s'"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:105
#, python-format
msgid "Invalid pool type : %(invalid_type)sExpected values : %(expected_values)s"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:110
#, python-format
msgid "Allocation pool must be empty for '%s' pool type"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:113
#, python-format
msgid "Allocation pool must have some value for '%s' pool type"
msgstr ""

#: eon/api/v1/esx_proxy_ip_pool.py:220
#, fuzzy, python-format
msgid "Unable to display the ip pool details.got the error : %s"
msgstr "无法显示 vCenter 详细信息。出现错误: %s"

#: eon/api/v1/esx_proxy_ip_pool.py:265
msgid "Enter appropriate IP range"
msgstr ""

#: eon/api/v1/managed_entity.py:45
msgid "The maximum number of clusters supported per proxy vm"
msgstr "每个代理 VM 支持的最大群集数"

#: eon/api/v1/managed_entity.py:157
#, python-format
msgid "Unable to display the resource:got the error : %s"
msgstr "无法显示资源: 出现错误: %s"

#: eon/api/v1/managed_entity.py:273
#, fuzzy, python-format
msgid "Unable to import the resource  (%(resource_name)s) Got error: %(error)s"
msgstr "无法导入资源(%s)，出现错误:%s"

#: eon/api/v1/managed_entity.py:330
msgid "Virtual machines are already running on this cluster."
msgstr ""

#: eon/api/v1/managed_entity.py:332
msgid ""
"Virtual machine instances are running on the compute node. Manually "
"remove these instances using the same tools that you used to manage the "
"instances. Instances consume resources and may cause oversubscription "
"when not managed by CloudSystem"
msgstr ""

#: eon/api/v1/managed_entity.py:395
#, python-format
msgid "Cluster '%s' does not have any hosts in connected state"
msgstr ""

#: eon/api/v1/managed_entity.py:397
msgid "Add host to the cluster or reconnect the disconnected hosts"
msgstr ""

#: eon/api/v1/managed_entity.py:402
#, python-format
msgid ""
"Could not determine vShield vApp status on hosts [%(hosts_without_vapp)s]"
" in cluster '%(cluster_name)s'"
msgstr ""

#: eon/api/v1/managed_entity.py:407
msgid "vShield vApp must be installed and running each host of the cluster"
msgstr ""

#: eon/api/v1/managed_entity.py:416
#, python-format
msgid "Could not determine if the cluster '%s' has hosts in connected state"
msgstr ""

#: eon/api/v1/managed_entity.py:438
msgid "Another cluster activation is in progress."
msgstr ""

#: eon/api/v1/managed_entity.py:439
msgid "Wait for the current activation task to complete and try again"
msgstr ""

#: eon/api/v1/managed_entity.py:460
#, fuzzy, python-format
msgid "Could not find Cluster %s"
msgstr "找不到 vCenter %s %s"

#: eon/api/v1/managed_entity.py:461
#, fuzzy, python-format
msgid "Import the cluster"
msgstr "群集的属性:%s"

#: eon/api/v1/managed_entity.py:468
#, fuzzy, python-format
msgid "Cluster %s is already activated or in the process of activation"
msgstr "已激活或正在激活资源名称 %s"

#: eon/api/v1/managed_entity.py:470
msgid ""
"Wait for the current cluster activation task to complete before retrying "
"activation"
msgstr ""

#: eon/api/v1/managed_entity.py:497
#, fuzzy, python-format
msgid ""
"The switch having access to cloud trunk is not configured for the cluster"
" %(cluster_name)s in datacenter %(dc)s"
msgstr "没有为群集 %s 配置交换机，数据中心= %s"

#: eon/api/v1/managed_entity.py:503
msgid "Configure the cloud trunk network switch."
msgstr ""

#: eon/api/v1/managed_entity.py:526
#, python-format
msgid ""
"The cluster '%(cluster_name)s' configuration does not meet the "
"requirements for activation. Errors = %(errors)s"
msgstr ""

#: eon/api/v1/managed_entity.py:531
#, python-format
msgid ""
"The cluster must be DRS enabled, have hosts in connected state, at least "
"one shared datastore and all hosts have access to the switch configured "
"for data trunk networking. The switch '%(switch_name)s' is configured for"
" the datacenter '%(dc)s'"
msgstr ""

#: eon/api/v1/managed_entity.py:584
#, fuzzy, python-format
msgid "Unable to activate the cluster %(cluster_name)s. Internal error occurred"
msgstr "无法激活群集 %s，出现错误:%s"

#: eon/api/v1/managed_entity.py:667
#, python-format
msgid ""
"Internal error occurred while updating the compute node state in DB. "
"[Error=%s]"
msgstr ""

#: eon/api/v1/managed_entity.py:720 eon/api/v1/managed_entity.py:764
#, fuzzy, python-format
msgid "Unable to activate the cluster %(cluster_name)s Got the error : %(error)s"
msgstr "无法激活群集 %s，出现错误:%s"

#: eon/api/v1/managed_entity.py:785
msgid "The compute node is activated and cannot be deleted."
msgstr ""

#: eon/api/v1/managed_entity.py:787
msgid "Deactivate the compute node and retry delete."
msgstr ""

#: eon/api/v1/managed_entity.py:812
#, python-format
msgid "Cannot remove resource (%s) in since it is not imported."
msgstr "无法删除资源(%s)，因为未导入该资源。"

#: eon/api/v1/managed_entity.py:828
#, fuzzy, python-format
msgid ""
"Not deleting the proxy vm. Since there are active resource(s) associated "
"to the proxy (%s)"
msgstr "无法删除! 具有与 vCenter (%s)关联的活动资源"

#: eon/api/v1/managed_entity.py:839
#, fuzzy, python-format
msgid "Could not delete the compute proxy %s"
msgstr "无法检索 vCenter %s"

#: eon/api/v1/managed_entity.py:841
#, python-format
msgid "Delete the proxy vm %s if exists and try to activate again."
msgstr ""

#: eon/api/v1/managed_entity.py:914
msgid "Another cluster deactivation is in progress."
msgstr ""

#: eon/api/v1/managed_entity.py:915
msgid "Wait for the current deactivation task to complete and try again."
msgstr ""

#: eon/api/v1/managed_entity.py:1055
msgid "The deactivation has been initiated."
msgstr ""

#: eon/api/v1/managed_entity.py:1062
#, fuzzy, python-format
msgid "Failed to deactivate cluster. Got error: %s"
msgstr "无法激活群集 %s，出现错误:%s"

#: eon/api/v1/vcenter.py:205
#, python-format
msgid "Error while listing the vCenters : %s"
msgstr "在列出 vCenter 时出错: %s"

#: eon/api/v1/vcenter.py:279
#, python-format
msgid "Unable to display the vcenter details.got the error : %s"
msgstr "无法显示 vCenter 详细信息。出现错误: %s"

#: eon/api/v1/vcenter.py:331
msgid "Registration/update of vCenter failed."
msgstr ""

#: eon/api/v1/vcenter.py:332
msgid ""
"Enter valid vCenter credentials and ensure vCenter is running and is "
"reachable from the appliance"
msgstr ""

#: eon/api/v1/vcenter.py:357
#, python-format
msgid ""
"Failed to register vCenter. The vCenter IP Address (%s) could not be "
"resolved."
msgstr ""

#: eon/api/v1/vcenter.py:370
#, fuzzy, python-format
msgid "Failed to retrieve data for vCenter %s"
msgstr "无法检索 vCenter %s"

#: eon/api/v1/vcenter.py:378
#, python-format
msgid "Failed to register vCenter. vCenter by ip_address (%s) already exists."
msgstr ""

#: eon/api/v1/vcenter.py:404
#, fuzzy, python-format
msgid ""
"There are cluster(s) activated and therefore the vShield credentials "
"cannot be deleted"
msgstr "数据中心 %s 使用不同的交换机名称(%s)激活了群集，因此，无法修改交换机名称"

#: eon/api/v1/vcenter.py:419 eon/api/v1/vcenter.py:780
msgid "All vShield credentials must be specified"
msgstr "必须指定所有 vShield 凭据"

#: eon/api/v1/vcenter.py:452
#, python-format
msgid "vCenter with id [%s] does not exist"
msgstr "具有 ID [%s] 的 vCenter 不存在"

#: eon/api/v1/vcenter.py:461
#, python-format
msgid "Update of vCenter failed: vCenter ip_address (%s) could not be resolved."
msgstr ""

#: eon/api/v1/vcenter.py:472
#, python-format
msgid "Could not retrieve vCenter %s"
msgstr "无法检索 vCenter %s"

#: eon/api/v1/vcenter.py:482
#, python-format
msgid "Registration/update of vCenter failed: vCenter (%s) already exists."
msgstr ""

#: eon/api/v1/vcenter.py:537
#, fuzzy, python-format
msgid ""
"The datacenter %(dc)s has cluster(s) activated with different switch name"
" (%(switch_details)s) and therefore the switch name cannot be modified"
msgstr "数据中心 %s 使用不同的交换机名称(%s)激活了群集，因此，无法修改交换机名称"

#: eon/api/v1/vcenter.py:553
#, fuzzy, python-format
msgid "There are activated/imported clusters(s) associated to the vCenter (ID %s)"
msgstr "无法删除! 具有与 vCenter (%s)关联的活动资源"

#: eon/api/v1/vcenter.py:621
#, python-format
msgid "Error validating vShield credentials : %s"
msgstr "在验证 vShield 凭据时出错: %s"

#: eon/api/v1/vcenter.py:622
#, fuzzy
msgid "Enter valid credentials"
msgstr "无法验证 vshield 凭据"

#: eon/api/v1/vcenter.py:629
#, fuzzy, python-format
msgid "vShield IP %(ip_address)s is invalid for vCenter %(vcenter_ip)s "
msgstr "vShield IP %s 对于 vCenter %s 无效"

#: eon/api/v1/vcenter.py:633
msgid "Enter valid VShield IP address"
msgstr ""

#: eon/api/v1/vcenter.py:830 eon/api/v1/vcenter.py:840
#, fuzzy, python-format
msgid "Registering of VCenter failed. Got error: '%s'"
msgstr "正在注册 vCenter，上下文:%s"

#: eon/api/v1/vcenter.py:967 eon/api/v1/vcenter.py:976
#, python-format
msgid "Failed to update vCenter. Got error: %s"
msgstr "无法更新 vCenter。出现错误:%s"

#: eon/api/v1/vcenter.py:1027
#, fuzzy, python-format
msgid "Failed to unregister vCenter.Got error: %s"
msgstr "无法更新 vCenter。出现错误:%s"

#: eon/api/v1/vcenter.py:1035
#, fuzzy, python-format
msgid "Failed to unregister vCenter. Got error: %s"
msgstr "无法更新 vCenter。出现错误:%s"

#: eon/common/config.py:24
msgid ""
"Partial name of a pipeline in your paste configuration file with the "
"service name removed. For example, if your paste section name is "
"[pipeline:isc-api-keystone] use the value \"keystone\""
msgstr ""
"粘贴配置文件中删除了服务名称的部分管道名称。例如，如果粘贴部分名称为 [pipeline:isc-api-"
"keystone]，则使用值“keystone”"

#: eon/common/config.py:30
msgid "Name of the paste configuration file."
msgstr "粘贴配置文件的名称。"

#: eon/common/config.py:34
msgid "Python module path of data access API"
msgstr "数据访问 API 的 Python 模块路径"

#: eon/common/config.py:36
msgid ""
"Default value for the number of items returned by a request if not "
"specified explicitly in the request"
msgstr "未在请求中明确指定时，请求返回的项目数的默认值"

#: eon/common/config.py:39
msgid "Maximum permissible number of items that could be returned by a request"
msgstr "请求可返回的最大允许项目数"

#: eon/common/config.py:42
msgid "Deploy the v1 OpenStack Images API. "
msgstr "部署 v1 OpenStack 映像 API。"

#: eon/common/config.py:44
msgid "The hostname/IP of the pydev process listening for debug connections"
msgstr "侦听调试连接的 pydev 进程的主机名/IP"

#: eon/common/config.py:47
msgid "The port on which a pydev process is listening for connections."
msgstr "pydev 进程侦听连接时使用的端口。"

#: eon/common/config.py:50
msgid ""
"Key used for encrypting sensitive metadata while talking to the registry "
"or database."
msgstr "与注册表或数据库通信时，用于加密敏感元数据的密钥。"

#: eon/common/config.py:78
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr ""

#: eon/common/config.py:96
msgid "Invalid syslog facility"
msgstr "syslog 工具无效"

#: eon/common/config.py:134
#, fuzzy, python-format
msgid "Unable to locate paste config file for %s."
msgstr "粘贴配置文件的名称。"

#: eon/common/config.py:172
#, python-format
msgid ""
"Unable to load %(app_name)s from configuration file %(conf_file)s.\n"
"Got: %(e)r"
msgstr ""
"无法加载 %(app_name)s (从配置文件 %(conf_file)s 中)。\n"
"获取的是:%(e)r"

#: eon/common/exception.py:25
msgid "An unknown exception occurred"
msgstr "发生未知异常"

#: eon/common/exception.py:50
msgid "Missing required argument."
msgstr "缺少所需的参数。"

#: eon/common/exception.py:54
#, python-format
msgid "Missing required credential: %(required)s"
msgstr "缺少所需的凭据:%(required)s"

#: eon/common/exception.py:58
#, python-format
msgid ""
"Incorrect auth strategy, expected \"%(expected)s\" but received "
"\"%(received)s\""
msgstr "不正确的身份验证策略，应为 \"%(expected)s\"，但收到 \"%(received)s\""

#: eon/common/exception.py:63
msgid "An object with the specified identifier was not found."
msgstr "找不到具有指定标识符的对象。"

#: eon/common/exception.py:67
#, python-format
msgid "Unknown scheme '%(scheme)s' found in URI"
msgstr "在 URI 中找到未知方案“%(scheme)s”"

#: eon/common/exception.py:71
msgid "The Store URI was malformed."
msgstr "存储 URI 的格式不正确。"

#: eon/common/exception.py:75
msgid "An object with the same identifier already exists."
msgstr "具有相同标识符的对象已存在。"

#: eon/common/exception.py:79
#, python-format
msgid "Connect error/bad request to Auth service at URL %(url)s."
msgstr "将错误/不正确的请求连接到 URL %(url)s 处的授权服务。"

#: eon/common/exception.py:83
#, python-format
msgid "Auth service at URL %(url)s not found."
msgstr "找不到 URL %(url)s 处的授权服务。"

#: eon/common/exception.py:87
msgid "Authorization failed."
msgstr "授权失败。"

#: eon/common/exception.py:91
msgid "You are not authenticated."
msgstr "您未进行身份验证。"

#: eon/common/exception.py:95 eon/common/exception.py:100
msgid "You are not authorized to complete this action."
msgstr "您无权完成此操作。"

#: eon/common/exception.py:104
msgid "Data supplied was not valid."
msgstr "提供的数据无效。"

#: eon/common/exception.py:108
msgid "Sort key supplied was not valid."
msgstr "提供的排序键无效。"

#: eon/common/exception.py:112
msgid "Unable to filter using the specified range."
msgstr "无法使用指定的范围进行过滤。"

#: eon/common/exception.py:116
#, python-format
msgid "Attribute '%(property)s' is read-only."
msgstr "属性“%(property)s”是只读的。"

#: eon/common/exception.py:120
#, python-format
msgid "Attribute '%(property)s' is reserved."
msgstr "属性“%(property)s”是保留的。"

#: eon/common/exception.py:124
#, python-format
msgid "Redirecting to %(uri)s for authorization."
msgstr "正在重定向到 %(uri)s 以进行授权。"

#: eon/common/exception.py:128
msgid "There was an error migrating the database."
msgstr "在迁移数据库时出错。"

#: eon/common/exception.py:132
msgid "There was an error connecting to a server"
msgstr "在连接到服务器时出错"

#: eon/common/exception.py:136
msgid "There was an error configuring the client."
msgstr "在配置客户端时出错。"

#: eon/common/exception.py:140
#, python-format
msgid ""
"The request returned a 302 Multiple Choices. This generally means that "
"you have not included a version indicator in a request URI.\n"
"\n"
"The body of response returned:\n"
"%(body)s"
msgstr ""
"请求返回了 302 多个选项。这通常意味着，未在请求 URI 中包含版本指示符。\n"
"\n"
"返回的响应正文:\n"
"%(body)s"

#: eon/common/exception.py:146
#, python-format
msgid ""
"The request returned a 413 Request Entity Too Large. This generally means"
" that rate limiting or a quota threshold was breached.\n"
"\n"
"The response body:\n"
"%(body)s"
msgstr ""
"请求返回了 413 请求实体太大。这通常意味着，超过了速率限制或配额阈值。\n"
"\n"
"响应正文:\n"
"%(body)s"

#: eon/common/exception.py:157
#, fuzzy
msgid ""
"The request returned 503 Service Unavailable. This generally occurs on "
"service overload or other transient outage."
msgstr "请求返回了 503 服务不可用。在服务过载或其它暂时性中断时，通常会出现该问题。"

#: eon/common/exception.py:168
msgid "The request returned 500 Internal Server Error."
msgstr "请求返回了 500 内部服务器错误。"

#: eon/common/exception.py:172
#, python-format
msgid ""
"The request returned an unexpected status: %(status)s.\n"
"\n"
"The response body:\n"
"%(body)s"
msgstr ""
"请求返回了意外状态:%(status)s。\n"
"\n"
"响应正文:\n"
"%(body)s"

#: eon/common/exception.py:177
#, python-format
msgid "Invalid content type %(content_type)s"
msgstr "无效的内容类型 %(content_type)s"

#: eon/common/exception.py:181
#, python-format
msgid "Registry was not configured correctly on API server. Reason: %(reason)s"
msgstr "未在 API 服务器上正确配置注册表。原因:%(reason)s"

#: eon/common/exception.py:186
#, python-format
msgid "Store %(store_name)s could not be configured correctly. Reason: %(reason)s"
msgstr "无法正确配置存储 %(store_name)s。原因:%(reason)s"

#: eon/common/exception.py:191
#, python-format
msgid ""
"Driver %(driver_name)s could not be configured correctly. Reason: "
"%(reason)s"
msgstr "无法正确配置驱动程序 %(driver_name)s。原因:%(reason)s"

#: eon/common/exception.py:196
#, python-format
msgid "Maximum redirects (%(redirects)s) was exceeded."
msgstr "超过最大重定向数(%(redirects)s)。"

#: eon/common/exception.py:200 eon/pavmm/utils.py:442
msgid "Error in generating the required certs for compute"
msgstr ""

#: eon/common/exception.py:204
msgid "Received invalid HTTP redirect."
msgstr "收到无效的 HTTP 重定向。"

#: eon/common/exception.py:208
msgid "Response from Keystone does not contain a isc endpoint."
msgstr "来自 Keystone 的响应不包含 ISC 端点。"

#: eon/common/exception.py:212
#, python-format
msgid "Server worker creation failed: %(reason)s."
msgstr "服务器工作线程创建失败:%(reason)s。"

#: eon/common/exception.py:216
#, python-format
msgid "Unable to load schema: %(reason)s"
msgstr "无法加载架构:%(reason)s"

#: eon/common/exception.py:220
#, python-format
msgid "Provided object does not match schema '%(schema)s': %(reason)s"
msgstr "提供的对象与架构“%(schema)s”不匹配:%(reason)s"

#: eon/common/exception.py:225
#, python-format
msgid "Provided header feature is unsupported: %(feature)s"
msgstr "不支持提供的标头功能:%(feature)s"

#: eon/common/exception.py:229
#, python-format
msgid "%(cls)s exception was raised in the last rpc call: %(val)s"
msgstr "在上次 RPC 调用中引发 %(cls)s 异常:%(val)s"

#: eon/common/exception.py:233 eon/common/exception.py:237
#, python-format
msgid "%(reason)s"
msgstr ""

#: eon/common/exception.py:246
#, python-format
msgid "Unable to store password: %(status)s"
msgstr ""

#: eon/common/exception.py:250
#, python-format
msgid "Unable to fetch password: %(status)s"
msgstr ""

#: eon/common/exception.py:259
#, python-format
msgid "The cluster could not be deactivated as a compute resource : %(reason)s"
msgstr ""

#: eon/common/exception.py:264
#, fuzzy, python-format
msgid "The cluster activation Failed : %(reason)s"
msgstr "服务器工作线程创建失败:%(reason)s。"

#: eon/common/exception.py:313
msgid "Invalid Operation"
msgstr ""

#: eon/common/exception.py:322
msgid "Failed to upgrade Proxy."
msgstr ""

#: eon/common/exception.py:331
#, python-format
msgid "Database error has occurred [%(reason)s]."
msgstr ""

#: eon/common/exception.py:335
#, python-format
msgid "Proxy VM has errors: %(reason)s"
msgstr ""

#: eon/common/log.py:88
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr "对过时的配置的调用出现严重错误:%(msg)s"

#: eon/common/messageutils.py:69
msgid "Exception in . Retry                     counter:"
msgstr ""

#: eon/common/messageutils.py:78
#, python-format
msgid "Initiating status check on proxy %s"
msgstr ""

#: eon/common/utils.py:90
msgid "Token is not specified or is not valid"
msgstr "未指定令牌或无效"

#: eon/common/utils.py:145
msgid "SDN controller is not up / agent services are not running Trying again..."
msgstr ""

#: eon/common/utils.py:180
msgid "Checking the proxy vm status..."
msgstr ""

#: eon/common/utils.py:184
#, python-format
msgid "%s consumer is not responding"
msgstr ""

#: eon/common/utils.py:222 eon/common/utils.py:253
msgid "Can not get token/tenant id for the REST API"
msgstr ""

#: eon/common/utils.py:229
#, python-format
msgid "Unable to do %s. The cluster could not be updated."
msgstr ""

#: eon/common/utils.py:241
msgid "SDN agents are not running / not responding"
msgstr ""

#: eon/common/utils.py:245
#, python-format
msgid "The cluster %s has been successful."
msgstr ""

#: eon/common/utils.py:260
#, fuzzy, python-format
msgid "Unable to deactivate the cluster %s from the compute service."
msgstr "无法激活群集 %s，出现错误:%s"

#: eon/common/utils.py:263
#, python-format
msgid "Cluster %s Successfully removed from the compute service"
msgstr ""

#: eon/common/utils.py:386
#, fuzzy, python-format
msgid "Checking the Rabbitmq server status %s"
msgstr "rabbitmq 服务器的主机名"

#: eon/common/utils.py:389
msgid "Listing rabbitmq proxy users"
msgstr ""

#: eon/common/utils.py:392
#, python-format
msgid "The Rabbitmq server status %s"
msgstr ""

#: eon/common/utils.py:393
#, python-format
msgid "The rabbitmq proxy users %s"
msgstr ""

#: eon/common/utils.py:394
#, python-format
msgid "Listing the states of the nova services %s"
msgstr ""

#: eon/common/utils.py:397
#, python-format
msgid "Appliance-proxy state is improper. Got the exception%s"
msgstr ""

#: eon/common/utils.py:426
#, python-format
msgid ""
"Error validating vShield App configuration on the hosts of the cluster. "
"Error detail: %s"
msgstr ""

#: eon/common/utils.py:428
msgid ""
"Verify is the vShield manager is running and is accessible. vSheild "
"manager is used to determine if vApp is running on the hosts of the "
"cluster"
msgstr ""

#: eon/common/wsgi.py:39
msgid ""
"Address to bind the server.  Useful when selecting a particular network "
"interface."
msgstr "将服务器绑定到的地址。在选择特定网络接口时非常有用。"

#: eon/common/wsgi.py:42
msgid "The port on which the server will listen."
msgstr "服务器将侦听的端口。"

#: eon/common/wsgi.py:47
msgid "The backlog value that will be used when creating the TCP listener socket."
msgstr "在创建 TCP 侦听程序套接字时使用的积压工作值。"

#: eon/common/wsgi.py:50
msgid ""
"The value for the socket option TCP_KEEPIDLE.  This isthe time in seconds"
" that the connection must be idle before TCP starts sending keepalive "
"probes."
msgstr "套接字选项 TCP_KEEPIDLE 的值。这是 TCP 开始发送保持连接探测之前连接处于空闲状态的时间(秒)。"

#: eon/common/wsgi.py:53
msgid "CA certificate file to use to verify connecting clients."
msgstr "用于验证连接客户端的 CA 证书文件。"

#: eon/common/wsgi.py:55
msgid "Certificate file to use when starting API server securely."
msgstr "在安全地启动 API 服务器时使用的证书文件。"

#: eon/common/wsgi.py:57
msgid "Private key file to use when starting API server securely."
msgstr "在安全地启动 API 服务器时使用的私钥文件。"

#: eon/common/wsgi.py:62
msgid ""
"The number of child process workers that will be created to service API "
"requests."
msgstr "为处理 API 请求而创建的子进程工作线程数。"

#: eon/common/wsgi.py:164
msgid "You must implement __call__"
msgstr ""

#: eon/common/wsgi.py:219
#, python-format
msgid "Loading %(app_name)s from %(conf_file)s"
msgstr "正在加载 %(app_name)s (从 %(conf_file)s 中)"

#: eon/common/wsgi.py:318
msgid "Malformed JSON in request body."
msgstr "请求正文中的 JSON 的格式不正确。"

#: eon/db/sqlalchemy/api.py:41
msgid ""
"Period in seconds after which SQLAlchemy should reestablish its "
"connection to the database."
msgstr "SQLAlchemy 重新建立到数据库的连接之前等待的时间(秒)。"

#: eon/db/sqlalchemy/api.py:44
msgid "The number of times to retry a connection to the SQLserver."
msgstr "连接到 SQL 服务器的重试次数。"

#: eon/db/sqlalchemy/api.py:47
msgid ""
"The amount of time to wait (in seconds) before attempting to retry the "
"SQL connection."
msgstr "在尝试重试 SQL 连接之前等待的时间(秒)。"

#: eon/db/sqlalchemy/api.py:50
msgid "A boolean that determines if the database will be automatically created."
msgstr "确定是否自动创建数据库的布尔值。"

#: eon/db/sqlalchemy/api.py:144
#, python-format
msgid ""
"Error configuring registry database with supplied sql_connection. Got "
"error: %s"
msgstr "使用提供的 SQL 连接配置注册表数据库时出错。出现错误:%s"

#: eon/db/sqlalchemy/api.py:313 eon/tests/unit/db/simple/api.py:548
#, python-format
msgid "No vCenter found with ID %s"
msgstr "找不到具有 ID %s 的 vCenter"

#: eon/db/sqlalchemy/api.py:428 eon/tests/unit/db/simple/api.py:407
msgid "IP is in use. Can't delete it"
msgstr ""

#: eon/db/sqlalchemy/api.py:456 eon/tests/unit/db/simple/api.py:219
#: eon/tests/unit/db/simple/api.py:225
#, fuzzy, python-format
msgid "No ESX Proxy IP found with ID %s"
msgstr "找不到具有 ID %s 的 vCenter"

#: eon/db/sqlalchemy/api.py:530
#, fuzzy, python-format
msgid "IP %s already exists!"
msgstr "具有名称(%s)的 ESX 代理已存在"

#: eon/db/sqlalchemy/api.py:591 eon/tests/unit/db/simple/api.py:312
#: eon/tests/unit/db/simple/api.py:319
#, python-format
msgid "No ESX Proxy IP Pool found with ID %s"
msgstr ""

#: eon/db/sqlalchemy/api.py:628 eon/tests/unit/db/simple/api.py:372
msgid "Can't change pool type.IP already in use."
msgstr ""

#: eon/db/sqlalchemy/api.py:655
msgid "Unable to create ippool"
msgstr ""

#: eon/db/sqlalchemy/api.py:672
#, python-format
msgid "IP %s assign to %s."
msgstr ""

#: eon/db/sqlalchemy/api.py:680
msgid "No IPs found."
msgstr ""

#: eon/db/sqlalchemy/api.py:682
msgid "No free IP."
msgstr ""

#: eon/db/sqlalchemy/api.py:712
#, python-format
msgid "No IP assign to %s."
msgstr ""

#: eon/db/sqlalchemy/api.py:728 eon/db/sqlalchemy/api.py:763
#, fuzzy
msgid "Resource-Entity ID already exists!"
msgstr "具有标识符的 vCenter 已存在!"

#: eon/db/sqlalchemy/api.py:800 eon/tests/unit/db/simple/api.py:682
#, fuzzy, python-format
msgid "No Resource found with ID %s"
msgstr "找不到具有 ID %s 的 vCenter"

#: eon/db/sqlalchemy/api.py:869 eon/db/sqlalchemy/api.py:905
#, fuzzy
msgid "ESX_PROXY ID already exists!"
msgstr "具有标识符的 ESX 代理已存在!"

#: eon/db/sqlalchemy/api.py:949
#, python-format
msgid "No Esx-Proxy found (ID %s)"
msgstr ""

#: eon/db/sqlalchemy/api.py:976
#, fuzzy, python-format
msgid "No Esx-Proxy found with name %s "
msgstr "找不到具有 ID %s 的 vCenter"

#: eon/db/sqlalchemy/api.py:1061
#, fuzzy, python-format
msgid "vCenter ID %s already exists!"
msgstr "具有标识符的 vCenter 已存在!"

#: eon/db/sqlalchemy/api.py:1175
#, python-format
msgid "Could not update property %s for resource %s"
msgstr ""

#: eon/db/sqlalchemy/migration.py:36
msgid "database is not under migration control"
msgstr "未对数据库进行迁移控制"

#: eon/db/sqlalchemy/migration.py:76
msgid "database is already under migration control"
msgstr "已对数据库进行迁移控制"

#: eon/pavmm/utils.py:123
#, fuzzy, python-format
msgid "Creating proxy: %s"
msgstr "正在删除属性 %s %s"

#: eon/pavmm/utils.py:148
#, python-format
msgid "Starting the proxy: %s"
msgstr ""

#: eon/pavmm/utils.py:166
#, fuzzy, python-format
msgid "Deleting proxy %s:"
msgstr "正在删除属性 %s %s"

#: eon/pavmm/utils.py:212
#, python-format
msgid "Generating certificates for Compute Proxy %s"
msgstr ""

#: eon/pavmm/utils.py:233
msgid "IP address for Proxy Appliance not available."
msgstr ""

#: eon/pavmm/utils.py:234
msgid "Set IPs (DHCP/Static) for vCenter Proxy Appliance"
msgstr ""

#: eon/pavmm/utils.py:350
msgid "Failed to copy the certificates"
msgstr ""

#: eon/pavmm/utils.py:352
#, python-format
msgid "Generating certificates for Compute Proxy %s completed"
msgstr ""

#: eon/pavmm/utils.py:395
#, python-format
msgid "Deleting certificates for the VM Proxy %s"
msgstr ""

#: eon/pavmm/utils.py:463
#, python-format
msgid "Unable to create Compute Proxy Appliance. %s"
msgstr ""

#: eon/pavmm/utils.py:481
#, python-format
msgid "Failed to Power-ON the proxy Appliance. %s"
msgstr ""

#: eon/pavmm/utils.py:490
#, python-format
msgid "Starting Proxy VM %s upgrade.hostname"
msgstr ""

#: eon/pavmm/utils.py:491
msgid "Waiting for the proxy to upgrade."
msgstr ""

#: eon/pavmm/utils.py:502
#, fuzzy, python-format
msgid "Proxy VM %s upgrade successfully completed.hostname"
msgstr "已成功创建 ESX 代理 VM %s"

#: eon/pavmm/utils.py:507 eon/pavmm/utils.py:512
#, python-format
msgid "Failed to upgrade Proxy VM %shostname"
msgstr ""

#: eon/pavmm/utils.py:522
msgid "Waiting for the nova compute service to start"
msgstr ""

#: eon/pavmm/utils.py:537
#, fuzzy, python-format
msgid "Nova compute service has started sucessfully"
msgstr "Nova 计算服务无法启动，错误:%(status)s"

#: eon/pavmm/utils.py:550
#, fuzzy, python-format
msgid "Nova compute service failed to start"
msgstr "Nova 计算服务无法启动，错误:%(status)s"

#: eon/pavmm/utils.py:551
msgid ""
"If DHCP is selected for vCenter proxy VM, ensure that DHCP service is "
"running on the network. If static IP is selected, then ensure that the "
"free IPs in the pool are not already in use."
msgstr ""

#: eon/tests/unit/db/simple/api.py:259
#, python-format
msgid "IP %s already exists!ipaddress"
msgstr ""

#: eon/tests/unit/vmware/fake_vmware_api.py:118
#, python-format
msgid "Property %(attr)s not set for the managed object %(name)s"
msgstr ""

#: eon/tests/unit/vmware/fake_vmware_api.py:408
#, python-format
msgid "Logging out a session that is invalid or already logged out: %s"
msgstr ""

#: eon/tests/unit/vmware/fake_vmware_api.py:425
msgid "Session Invalid"
msgstr ""

#: eon/vmware/cluster_validate.py:77
msgid "No shared datastore available in the cluster"
msgstr ""

#: eon/vmware/cluster_validate.py:87
msgid "Cluster DRS not enabled"
msgstr ""

#: eon/vmware/inventory_collector.py:67
#, python-format
msgid "Exception occurred while monitoring updates on vCenter got the error : %s"
msgstr ""

#: eon/vmware/inventory_collector.py:198
#, python-format
msgid "Could not validate cluster configuration. Internal error: %s"
msgstr ""

#: eon/vmware/inventory_collector.py:247
msgid "No switches configured on the host"
msgstr ""

#: eon/vmware/inventory_collector.py:267
#, python-format
msgid "Hosts %s in the cluster do not have vSwitch '%s'"
msgstr ""

#: eon/vmware/inventory_collector.py:324
#, python-format
msgid "Hosts %s in the cluster do not have access to the dvSwitch"
msgstr ""

#: eon/vmware/inventory_collector.py:334
#, python-format
msgid ""
"The switch with name '%s' is configured as the data networking access "
"switch for this datacenter and the hosts in the cluster are not connected"
" to this switch"
msgstr ""

#: eon/vmware/inventory_collector.py:421
#, python-format
msgid "Update received on vCenter %s for MOR %s with property %s and type %s"
msgstr ""

#: eon/vmware/inventory_collector.py:439
#, python-format
msgid ""
"Delete event received on vCenter %(vcenter) for MOR "
"%(mor)vcenterip_address"
msgstr ""

#: eon/vmware/vim.py:121
#, python-format
msgid "Unable to validate session %s!"
msgstr ""

#: eon/vmware/vim_session.py:100
#, python-format
msgid "Could not find (%s) in VIM Object"
msgstr ""

#: eon/vmware/vim_session.py:115
#, python-format
msgid "Task : %s (%s) status: success"
msgstr ""

#: eon/vmware/vim_session.py:120
#, python-format
msgid "Task : %s (%s) status: error, cause: %s"
msgstr ""

#: eon/vmware/vim_util.py:61
msgid "urllib2 error "
msgstr ""

#: eon/vmware/vim_util.py:67
msgid "urllib2 error : "
msgstr ""

#: eon/vmware/vim_util.py:81
msgid "Socket timeout"
msgstr ""

#: eon/vmware/vim_util.py:94
msgid "Exception"
msgstr ""

#: eon/vmware/vim_util.py:105
#, python-format
msgid "Error(s) %s occurred in the call to RetrieveProperties"
msgstr ""

#: eon/vmware/vshield_client.py:73
#, python-format
msgid "Error in connecting to vShield manager %s"
msgstr ""

#: eon/vmware/vshield_client.py:76
#, python-format
msgid "An error occurred while executing vShield API: %s"
msgstr ""

#: eon/vmware/vshield_client.py:85
#, fuzzy
msgid "Could not validate vShield credentials"
msgstr "无法验证 vshield 凭据"

#~ msgid "Unable to retrieve request id from context"
#~ msgstr "无法从上下文中检索请求 ID"

#~ msgid "Determining version of request: %(method)s %(path)s Accept: %(accept)s"
#~ msgstr "正在确定请求版本:%(method)s %(path)s 接受:%(accept)s"

#~ msgid "Using url versioning"
#~ msgstr "使用 URL 版本控制"

#~ msgid "Unknown version. Returning version choices."
#~ msgstr "未知版本。正在返回版本选项。"

#~ msgid "Matched version: v%d"
#~ msgstr "匹配的版本:v%d"

#~ msgid "Validating required attributes...Got: %s, Required attributes: %s"
#~ msgstr "正在验证所需的属性...获取的是:%s，所需的属性:%s"

#~ msgid "Fetching list of all registered ESX-Proxy"
#~ msgstr "正在获取注册的 ESX 代理列表"

#~ msgid "Validating ESX-Proxy before creating, ESX-Proxy: %s"
#~ msgstr "在创建之前，正在验证 ESX 代理，ESX 代理:%s"

#~ msgid "ESX-Proxy by name (%s) not found"
#~ msgstr "找不到具有名称 %s 的 ESX 代理"

#~ msgid "Validating ESX-Proxy before updating, ESX-Proxy id: %s"
#~ msgstr "在更新之前，正在验证 ESX 代理，ESX 代理 ID:%s"

#~ msgid "Validating ESX-Proxy(id: %s) before deleting"
#~ msgstr "在删除之前，正在验证 ESX 代理(ID:%s)"

#~ msgid "Failed to delete %s"
#~ msgstr "无法删除 %s"

#~ msgid "Registering ESX-Proxy, context: %s, ESX-Proxy data: %s"
#~ msgstr "正在注册 ESX 代理，上下文:%s，ESX 代理数据:%s"

#~ msgid "Required attributes validation failed: %s"
#~ msgstr "所需的属性验证失败:%s"

#~ msgid "Cannot create ESX-Proxy reason: %s"
#~ msgstr "无法创建 ESX 代理，原因:%s"

#~ msgid "Successfully created esx_proxy"
#~ msgstr "已成功创建 ESX 代理"

#~ msgid "Failed to add esx_proxy metadata. Got error: %(e)s"
#~ msgstr "无法添加 ESX 代理元数据。出现错误:%(e)s"

#~ msgid "Updating ESX-Proxy, context: %s"
#~ msgstr "正在更新 ESX 代理，上下文:%s"

#~ msgid "ESX-Proxy data: %s"
#~ msgstr "ESX 代理数据:%s"

#~ msgid "Required attribute validation failed: %s"
#~ msgstr "所需的属性验证失败:%s"

#~ msgid "Cannot update ESX-Proxy(ID: %s) reason: %s"
#~ msgstr "无法更新 ESX 代理(ID:%s)，原因:%s"

#~ msgid "Successfully updated esx_proxy"
#~ msgstr "已成功更新 ESX 代理"

#~ msgid "Failed to update vCenter. Got error: %(e)s"
#~ msgstr "无法更新 vCenter。出现错误:%(e)s"

#~ msgid "Deleting ESX-Proxy, context: %s id: %s"
#~ msgstr "正在删除 ESX 代理，上下文:%s ID:%s"

#~ msgid "Cannot delete ESX-Proxy(ID: %s) reason: %s"
#~ msgstr "无法删除 ESX 代理(ID:%s)，原因:%s"

#~ msgid "Successfully destroyed esx_proxy"
#~ msgstr "已成功销毁 ESX 代理"

#~ msgid "Failed to delete esx_proxy. Got error: %(e)s"
#~ msgstr "无法删除 ESX 代理。出现错误:%(e)s"

#~ msgid "Store for scheme %s not found"
#~ msgstr "找不到方案 %s 的存储"

#~ msgid "Unable to display the resources:got the error : %s"
#~ msgstr "无法显示资源，出现错误:%s"

#~ msgid "Registering resource, context: %s, vCenter: %s, Resource: %s"
#~ msgstr "正在注册资源，上下文:%s，vCenter:%s，资源:%s"

#~ msgid "Checking if max supported clusters limit reached"
#~ msgstr "正在检查是否达到最大支持的群集限制"

#~ msgid "Max supported clusters limit reached, Creating"
#~ msgstr "已达到最大支持的群集限制，正在创建"

#~ msgid "Using existing ESX-Proxy: %s"
#~ msgstr "使用现有的 ESX 代理:%s"

#~ msgid "state_change for resource: %s state = %s"
#~ msgstr "资源的状态更改:%s 状态 = %s"

#~ msgid "Import resource %s "
#~ msgstr "导入资源 %s"

#~ msgid "Imported resource %s and imported id is %s"
#~ msgstr "已导入资源 %s，导入的 ID 为 %s"

#~ msgid "Imported resource %s and sending response %s"
#~ msgstr "已导入资源 %s，正在发送响应 %s"

#~ msgid "Import resource information passed is %s"
#~ msgstr "传递的导入资源信息是 %s"

#~ msgid "Resource name %s is already importedresource_name"
#~ msgstr "已导入资源名称 %s"

#~ msgid "Resource name %s is not imported"
#~ msgstr "未导入资源名称 %s"

#~ msgid "Found imported resource %s"
#~ msgstr "找到导入的资源 %s"

#~ msgid "Found valid imported resource %s"
#~ msgstr "找到导入的有效资源 %s"

#~ msgid "DC for resource %s is %s"
#~ msgstr "资源 %s 的 DC 是 %s"

#~ msgid "cluster data = %s"
#~ msgstr "群集数据 = %s"

#~ msgid ""
#~ "Switch %s has configuration errors for"
#~ " the cluster %s in datacenter %s. "
#~ "Errors = %s"
#~ msgstr "交换机 %s 出现配置错误，群集= %s，数据中心= %s。错误 = %s"

#~ msgid " Resource entity details: %s "
#~ msgstr " 资源实体详细信息:%s "

#~ msgid "remove_resource %s %s %s"
#~ msgstr "删除资源 %s %s %s"

#~ msgid "Removed resource %s and sending response %s"
#~ msgstr "已删除资源 %s，正在发送响应 %s"

#~ msgid "Un-Registering resource id: %s"
#~ msgstr "正在取消注册资源 ID:%s"

#~ msgid "Successfully unregistered resource [%s]"
#~ msgstr "已成功取消注册资源 [%s]"

#~ msgid "Unable to unregister resource %sGot error: %s"
#~ msgstr "无法取消注册资源 %sGot，错误:%s"

#~ msgid "Successfully destroyed resource_entity"
#~ msgstr "已成功销毁资源实体"

#~ msgid "Failed to delete vCenter. Got error: %(e)s"
#~ msgstr "无法删除 vCenter。出现错误:%(e)s"

#~ msgid "ManagedEntitySerializer"
#~ msgstr "托管实体序列化程序"

#~ msgid "Fetching list of all registered vcenters"
#~ msgstr "正在获取所有注册的 vCenter 的列表"

#~ msgid "Unable to list the details of vCenter,Got error %s"
#~ msgstr "无法列出 vCenter 详细信息，出现错误 %s"

#~ msgid "Details of vcenter:  %s has been returned"
#~ msgstr "vCenter 详细信息:已返回 %s"

#~ msgid "ip_address"
#~ msgstr "IP 地址"

#~ msgid ""
#~ "Registration of vCenter failed.\n"
#~ "This could be due to invalid "
#~ "credentials or the vCenter server %s "
#~ "is unreachable.ip_address"
#~ msgstr ""
#~ "vCenter 注册失败。\n"
#~ "原因可能是凭据无效或无法访问 vCenter 服务器 %s 的 IP 地址"

#~ msgid "vCenter with id (%s) does not exist"
#~ msgstr "具有 ID (%s)的 vCenter 不存在"

#~ msgid "Check vCenter connectivity since IP has changed old IP = %s new IP = %s"
#~ msgstr "检查 vCenter 连接，因为 IP 已更改，旧 IP = %s，新 IP = %s"

#~ msgid "Configured settings = %s"
#~ msgstr "配置的设置 = %s"

#~ msgid ""
#~ "Fail to delete, there are active "
#~ "Resource(s) associated to vCenter (ID "
#~ "%s)"
#~ msgstr "无法删除，存在与 vCenter (ID %s) 关联的活动资源"

#~ msgid "The list of proxy associated to the vCenter %s"
#~ msgstr "与 vCenter %s 关联的代理列表"

#~ msgid "Fetch all resources associated to ESX-Proxy(ID: %s)"
#~ msgstr "获取与 ESX 代理(ID:%s)"

#~ msgid "Returning...Resource-Name: %s, Resource-Path: %s"
#~ msgstr "正在返回...资源名称:%s，资源路径:%s"

#~ msgid "Required attributes %s"
#~ msgstr "必需属性 %s"

#~ msgid "Successfully added the vcenter %s"
#~ msgstr "已成功添加 vCenter %s"

#~ msgid "Unable to display the vcenter details:got the error : %s"
#~ msgstr "无法显示 vCenter 详细信息，出现错误:%s"

#~ msgid "Updating vcenter, context: %s"
#~ msgstr "正在更新 vCenter，上下文:%s"

#~ msgid "vCenter-Data: name: %s, username: %s, ip_address: %s, port: %s"
#~ msgstr "vCenter 数据:名称:%s，用户名:%s，IP 地址:%s，端口:%s"

#~ msgid "Reconfigured switch settings = %s"
#~ msgstr "重新配置的交换机设置 = %s"

#~ msgid "Updating settings = %s"
#~ msgstr "正在更新设置 = %s"

#~ msgid "Successfully updated vcenter settings"
#~ msgstr "已成功更新 vCenter 设置"

#~ msgid "Response for update vCenter %s"
#~ msgstr "更新 vCenter %s 的响应"

#~ msgid "Unregister vcenter id: %s"
#~ msgstr "取消注册 vCenter ID:%s"

#~ msgid "Successfully deleted vcenter"
#~ msgstr "已成功删除 vCenter"

#~ msgid "Failed to delete vCenter. Got error: %s"
#~ msgstr "无法删除 vCenter。出现错误:%s"

#~ msgid "PAVMM Create VM REST api return error: %(status)s"
#~ msgstr "PAVMM 创建 VM REST API 返回错误:%(status)s"

#~ msgid "PAVMM Start VM REST api return error: %(status)s"
#~ msgstr "PAVMM 启动 VM REST API 返回错误:%(status)s"

#~ msgid "PAVMM Delete VM REST api return error: %(status)s"
#~ msgstr "PAVMM 删除 VM REST API 返回错误:%(status)s"

#~ msgid "Max serialization depth exceeded on object: %d %s"
#~ msgstr "已超过对象上的最大序列化深度:%d %s"

#~ msgid "Deprecated: %s"
#~ msgstr "已过时:%s"

#~ msgid "syslog facility must be one of: %s"
#~ msgstr "syslog 工具必须是以下值之一:%s"

#~ msgid "Request must be a list of commands"
#~ msgstr "请求必须是一个命令列表"

#~ msgid "Bad Command: %s"
#~ msgstr "错误的命令:%s"

#~ msgid "Wrong command structure: %s"
#~ msgstr "错误的命令结构:%s"

#~ msgid "Command not found"
#~ msgstr "找不到命令"

#~ msgid ""
#~ "RPC Call Error: %(val)s\n"
#~ "%(tb)s"
#~ msgstr ""
#~ "RPC 调用错误:%(val)s\n"
#~ "%(tb)s"

#~ msgid "Read-only access"
#~ msgstr "只读访问"

#~ msgid "Invalid backend: %s"
#~ msgstr "无效的后端:%s"

#~ msgid "The authorization key is not present"
#~ msgstr "授权密钥不存在"

#~ msgid "Authorization token is empty or not present"
#~ msgstr "授权令牌为空或不存在"

#~ msgid "Authorization check response is %s"
#~ msgstr "授权检查响应是 %s"

#~ msgid "Authorization check componentID is %s"
#~ msgstr "授权检查组件 ID 是 %s"

#~ msgid ""
#~ "When running server in SSL mode, "
#~ "you must specify both a cert_file "
#~ "and key_file option value in your "
#~ "configuration file"
#~ msgstr "在 SSL 模式下运行服务器时，必须在配置文件中指定 cert_file 和 key_file 选项值"

#~ msgid "Could not bind to %s:%s after trying for 30 seconds"
#~ msgstr "无法绑定到 %s:%s在尝试 30 秒后"

#~ msgid "Starting %d workers"
#~ msgstr "正在启动 %d 个工作线程"

#~ msgid "Removing dead child %s"
#~ msgstr "正在删除失效的子项 %s"

#~ msgid "Not respawning child %d, cannot recover from termination"
#~ msgstr "未重新生成子项 %d，无法从终止中恢复"

#~ msgid "All workers have terminated. Exiting"
#~ msgstr "已终止所有工作线程。正在退出"

#~ msgid "Caught keyboard interrupt. Exiting."
#~ msgstr "已捕获键盘中断。正在退出。"

#~ msgid "Exited"
#~ msgstr "已退出"

#~ msgid "Child %d exiting normally"
#~ msgstr "子项 %d 正在正常退出"

#~ msgid "Started child %s"
#~ msgstr "已启动子项 %s"

#~ msgid "eventlet 'poll' hub is not available on this platform"
#~ msgstr "eventlet 'poll' hub 在此平台上不可用"

#~ msgid "Starting single process server"
#~ msgstr "正在启动单个进程服务器"

#~ msgid "auto-creating isc registry DB"
#~ msgstr "正在自动创建 isc 注册表数据库"

#~ msgid "not auto-creating isc registry DB"
#~ msgstr "未自动创建 isc 注册表数据库"

#~ msgid "SQL connection failed. %d attempts left."
#~ msgstr "SQL 连接失败。还剩下 %d 次尝试。"

#~ msgid "Id not in sort_keys; is sort_keys unique?"
#~ msgstr "Id 不在 sort_keys 中；sort_keys 是唯一的吗?"

#~ msgid "Unknown sort direction, must be 'desc' or 'asc'"
#~ msgstr "未知排序方向，必须为“desc”或“asc”"

#~ msgid ""
#~ "get_properties_for_resource id = %s\n"
#~ " resource_id = %s\n"
#~ " name = %s\n"
#~ " type = %s\n"
#~ " value = %s"
#~ msgstr ""
#~ "get_properties_for_resource id = %s\n"
#~ " resource_id = %s\n"
#~ " name = %s\n"
#~ " type = %s\n"
#~ " value = %s"

#~ msgid ""
#~ "get_property_for_resource id = %s\n"
#~ " resource_id = %s\n"
#~ " name = %s\n"
#~ " type = %s\n"
#~ " value = %s"
#~ msgstr ""
#~ "get_property_for_resource id = %s\n"
#~ " resource_id = %s\n"
#~ " name = %s\n"
#~ " type = %s\n"
#~ " value = %s"

#~ msgid "update_resource_property %s %s %s %s"
#~ msgstr "update_resource_property %s %s %s %s"

#~ msgid "update_resource_properties %s %s"
#~ msgstr "update_resource_properties %s %s"

#~ msgid "update_resource_properties for %s %s returning %s"
#~ msgstr "%s %s 的 update_resource_properties，正在返回 %s"

#~ msgid "Method _delete_resource_property %s %s"
#~ msgstr "_delete_resource_property 方法 %s %s"

#~ msgid "Error deleting property %s %s"
#~ msgstr "在删除属性 %s %s 时出错"

#~ msgid "_delete_all_resource_properties %s"
#~ msgstr "_delete_all_resource_properties %s"

#~ msgid "delete_resource_properties %s %s"
#~ msgstr "delete_resource_properties %s %s"

#~ msgid "Upgrading database to version %s"
#~ msgstr "正在将数据库升级到版本 %s"

#~ msgid "Downgrading database to version %s"
#~ msgstr "正在将数据库降级到版本 %s"

#~ msgid "creating table %(table)s"
#~ msgstr "正在创建表 %(table)s"

#~ msgid "dropping table %(table)s"
#~ msgstr "正在删除表 %(table)s"

#~ msgid "The port on which the rabbitmq server is listening"
#~ msgstr "rabbitmq 服务器正在侦听的端口"

#~ msgid "A boolean value indicating if the selected rabbitmq server uses SSL."
#~ msgstr "指示选定 rabbitmq 服务器是否使用 SSL 的布尔值。"

#~ msgid "The user ID for authentication with rabbitmq."
#~ msgstr "通过 rabbitmq 进行身份验证的用户 ID。"

#~ msgid ""
#~ "The password that will be used for"
#~ " authentication with the rabbitmq server."
#~ msgstr "通过 rabbitmq 服务器进行身份验证时使用的密码。"

#~ msgid "The virtual host used in the rabbitmq connection."
#~ msgstr "rabbitmq 连接中使用的虚拟主机。"

#~ msgid "Exchange name to use for connection when using rabbit strategy."
#~ msgstr "使用 rabbit 策略时用于连接的交换名称。"

#~ msgid "Topic to use for connection when using rabbit strategy."
#~ msgstr "使用 rabbit 策略时用于连接的主题。"

#~ msgid "The maximum number of times to attempt to connect to the AMQP server."
#~ msgstr "尝试连接到 AMQP 服务器的最大次数。"

#~ msgid ""
#~ "This value multiplied by the number "
#~ "of connection attempts gives the amount"
#~ " of time in seconds to sleep "
#~ "between connection attempts to the AMQP"
#~ " server."
#~ msgstr "可以将该值乘以尝试的连接次数以得出尝试连接到 AMQP 服务器之间休眠的时间(秒)。"

#~ msgid ""
#~ "The maximum amount of time to wait"
#~ " between connection attempts.  The delay"
#~ " time will be the smaller of "
#~ "this value and the value of "
#~ "<rabbit_retry_backoff> * <the number of "
#~ "failed connection attempts so far>."
#~ msgstr ""
#~ "在连接尝试之间等待的最长时间。延迟时间是该值和 <rabbit_retry_backoff> * "
#~ "<the number of failed connection "
#~ "attempts so far> 值之间的较小值。"

#~ msgid "Reconnecting to rabbitmq...topic: %s"
#~ msgstr "正在重新连接到 rabbitmq...主题:%s"

#~ msgid "Attempting to publish message: %s"
#~ msgstr "正在尝试发布消息:%s"

#~ msgid "Message : %s has been published"
#~ msgstr "消息:已发布 %s"

#~ msgid "proc.stdout  :  %s"
#~ msgstr "proc.stdout:%s"

#~ msgid ""
#~ "Generating certificates & configurations... "
#~ "vCenter: %s, Resource-Name: %s, ESX-"
#~ "Proxy: %s, Routing-Key: %s, Host "
#~ "IP-Address: %s"
#~ msgstr "正在生成证书和配置... vCenter:%s，资源名称:%s，ESX 代理:%s，路由关键字:%s，主机 IP 地址:%s"

#~ msgid "Got neutron_password !"
#~ msgstr "已获取 neutron_password!"

#~ msgid "Rabbitmq creds created"
#~ msgstr "已创建 Rabbitmq 证书"

#~ msgid "All require directories created"
#~ msgstr "已创建所需的所有目录"

#~ msgid "Conf file written %s"
#~ msgstr "已写入配置文件 %s"

#~ msgid "Compute hostname: %s"
#~ msgstr "计算主机名:%s"

#~ msgid "RabbitMQ, Postgres certs created"
#~ msgstr "RabbitMQ，已创建 Postgres 证书"

#~ msgid "Deleted: Rabbitmq_user = %s"
#~ msgstr "已删除:Rabbitmq_user = %s"

#~ msgid "Creating VM...ESX-Proxy: %s, Resource: %s, vCenter: %s"
#~ msgstr "正在创建 VM...ESX 代理:%s，资源:%s，vCenter:%s"

#~ msgid "Writing temporary certificate...Compute node-name: %s, Config: %s"
#~ msgstr "正在写入临时证书...计算节点名称:%s，配置:%s"

#~ msgid "Compute Certs Location: %s "
#~ msgstr "计算证书位置:%s "

#~ msgid "unconfig cert for %s"
#~ msgstr "取消配置 %s 的证书"

#~ msgid "Setting PostgresConfig server-side config"
#~ msgstr "正在设置 PostgresConfig 服务器端配置"

#~ msgid "Setting PostgresConfig client-side config"
#~ msgstr "正在设置 PostgresConfig 客户端配置"

#~ msgid "Setting RabbitMQConfig server-side config"
#~ msgstr "正在设置 RabbitMQConfig 服务器端配置"

#~ msgid "Setting RabbitMQConfig client-side config"
#~ msgstr "正在设置 RabbitMQConfig 客户端配置"

#~ msgid "REQ URL: %s\n"
#~ msgstr "请求 URL:%s\n"

#~ msgid "REQ: %s\n"
#~ msgstr "请求:%s\n"

#~ msgid "REQ BODY: %s\n"
#~ msgstr "请求正文:%s\n"

#~ msgid "Error in executing vshield api for rest url %s, Cause : %s"
#~ msgstr "为 rest url %s 执行 vshield api 时出错，原因:%s"

#~ msgid "RESP:%s\n"
#~ msgstr "响应:%s\n"

#~ msgid "RESP BODY:%s\n"
#~ msgstr "响应正文:%s\n"

